app/src/repositories/comment/commentRepository.js

```diff
@@ -96,6 +96,17 @@ class CommentRepository {
       dislikeCount: Number(result.dislikeCount) || 0,
     };
   }
+
+  async getCommentsByEpisode(episodeId) {
+    const query = `
+    SELECT *
+    FROM comments
+    WHERE episode_id = ?
+    ORDER BY created_at ASC;
+    `;
+    const [rows] = await pool.query(query, [episodeId]);
+    return rows.map(toCamelCase);
```

Comment:
```suggestion
    return toCamelCase(rows);
```
toCamelCase 유틸 함수 내부에서 배열인 경우 map 변환을 이미 처리하고 있어서, `toCamelCase(rows)`로 호출하면 더 깔끔할 것 같아요!

Reply:
이 부분을 놓쳤네요! 알려주셔서 감사합니다 😀